arm/Microsoft.MachineLearningServices/workspaces/readme.md

```diff
@@ -39,6 +39,42 @@ This module deploys a Machine Learning Services Workspace.
 | `tags` | object | `{object}` |  | Optional. Resource tags. |
 | `workspaceId` | string |  |  | Optional. Resource ID of log analytics. |
 
+### Parameter Usage: `privateEndpoints`
```

Comment:
Double up. Remove one of the privateEndpoints

Reply:
done